zest/releaser/pypi.py

```diff
@@ -407,6 +407,31 @@ def version_levels(self):
             return default
         return result
 
+    def tag_format(self):
+        """Return the string formating the tag name in the release.
+
+        Override the default ``%(version)`` in setup.cfg using
+        a ``tag-format`` option::
+
+            [zest.releaser]
+            tag-format = v%(version)s
+
+        Returns default of ``%(version)s`` when nothing has been configured.
+        """
+        default = '%(version)s'
+        if self.config is None:
+            return default
+        try:
```

Comment:
I agree. So something like this:

```
try:
    result = self.config.get('zest.releaser', 'tag-format')
except (NoSectionError, NoOptionError, ValueError):
    return default
# And then continue with the checking.
```

BTW, `formater` should be spelled with two t's: `formatter`.